core/src/main/java/com/adobe/aio/workspace/Workspace.java

```diff
@@ -61,19 +66,23 @@ public class Workspace {
   private final String consumerOrgId;
   private final String projectId;
   private final String workspaceId;
+  private final String runtimeNamespace;
+  private final List<String> runtimeActions;
```

Comment:
could we remove `runtimeActions` from this workspace very static config
as it's will be created/deleted dynamically ?

Reply:
it's not used anyway in this open source sdk, is it ?

Reply:
correct, only `namespace` will be required. I will remove it

Reply:
done